.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       fail-fast: false
       matrix:
         go-version: [1.15.x, 1.16.x, 1.17.x]
-        libseccomp: ["v2.5.2", "HEAD"]
+        libseccomp: ["v2.2.1", "v2.2.3", "v2.3.3", "v2.4.3", "v2.5.2", "HEAD"]
```

Comment:
This is interesting as we no longer support v2.2.x, v2.3.x, or v2.4.x upstream anymore.  I guess we can let this slide for no so long as there are no major issues, but as soon as something significant crops up with these older releases we should probably just drop them.

Reply:
I'm merely following what's in there already (except for dropping 2.2.0 support).

If you want, I can rework this to drop 2.2.x support, too, as this is only used by very old (but still supported) Ubuntu 14.04. 
Dropping 2.3 and especially 2.4 is more problematic though, as this is what many long-term supported distros ship (Debian, RHEL, SLES).

I think it makes total sense to drop 2.2 and 2.3 (and maybe 2.4) _after_ the release is cut, but it's up to you to decide.